sdk/python/requirements.in

```
@@ -1,2 +1,3 @@
 kfp>=1.8.10,<1.8.19
-protobuf>=3.20.1,<4
\ No newline at end of file
+protobuf>=3.20.1,<4
+kfp-tekton-server-api==1.5.0
```

```suggestion
kfp-tekton-server-api>=1.5.0
```

This way is more flexible if we need to add bug patches to the api package.